src/alerts/version-alerts.ts

```diff
@@ -85,5 +85,13 @@ export const useTimeUntilRequiredUpdate = (): number | undefined => {
     }
   }, [updateRequiredBy, updateTimeRemaining]);
 
+  useEffect(() => {
```

Comment:
Was this supposed to be `onUnmount`?

Reply:
I prefer to use `useEffect` itself to take advantage of ESLint checking the dependency list. Terra UI has _so_ many bugs from `useOnMount` ignoring dependencies of the effect function.

Reply:
I agree, but the commit message said unmount instead of onmount, and I didn't think this code was an "unmount" thing. 

But I guess it's that the function that is returned is what is executed on unmount…. so I believe I understand.

Reply:
Ah. Yeah, since the `clearInterval` is in the cleanup function returned by the effect function, it runs when the hook is unmounted.